### PR TITLE
:bar_chart: migrate JSON data page for children stunting

### PR DESCRIPTION
**Pull request by Marigold**
Migrate [JSON-based Data page](https://ourworldindata.org/grapher/share-of-children-younger-than-5-who-suffer-from-stunting) to [ETL-based Data page](http://staging-site-mojmir/admin/datapage-preview/735978) by adding metadata to its grapher step. I tried to match the original Data page as closely as possible, which means some metadata fields might not **adhere to our guidelines**. I moved [FAQs to its GDoc](https://docs.google.com/document/d/1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw/edit) and used official topic tags names.

@lucasrodes could you please check the Data Page for any issues? You are also encouraged to update metadata in the YAML file according to guidelines. Commented lines are metadata from the ETL dataset, you should delete them if they're not helpful. **Add commits to this PR** as you wish!

You can test it by running
```
ENV=.env.mojmir GRAPHER_FILTER=sh_sta_stnt_zs etl grapher/worldbank_wdi/2023-05-29/wdi --grapher
```
and visiting the [data page](http://staging-site-mojmir/admin/datapage-preview/735978).

Parent issue https://github.com/owid/etl/issues/1666

### CODE REVIEW

File: etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
+dataset:
+  update_period_days: 365
+  sources: []
+definitions:
+  common:
+    sources: []
+    origins:
+      - producer: World Bank World Development Indicators
+        title: UNICEF, World Health Organization and World Bank
```
Comment (lucasrodes) [inline etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml:9]: If multiple sources are being listed, we should probably separate them using `;`

File: etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        description: |-
+          The World Development Indicators (WDI) is the World Bank’s primary collection of development indicators, compiled from officially-recognized international sources. It presents the most current and accurate global development data available and includes national, regional, and global estimates.
+
+          \[Text from [World Bank data catalog](https://datacatalog.worldbank.org/dataset/world-development-indicators)\]
```
Comment (lucasrodes) [inline etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml:13]: What's the purpose of the `\[`? Is it supposed to render differently on the data page?

image

File: etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        #   to two years old height is measured by recumbent length. For older children height is measured by stature while
+        #   standing. The data are based on the WHO's new child growth standards released in 2006.
+        description_key:
+          - Stunting is when a child is significantly shorter than the average for their age.
+          - Stunted growth is a consequence of poor nutrition and/or repeated infection.
+        description_from_producer: |-
+          Prevalence of stunting is the percentage of children under age 5 whose height for age is more than two standard deviations below the median for the international reference population ages 0-59 months. For children up to two years old height is measured by recumbent length. For older children height is measured by stature while standing. The data are based on the WHO's new child growth standards released in 2006.
+
+
+          \[Text from World Bank World Development Indicators [metadata glossary](https://databank.worldbank.org/metadataglossary/world-development-indicators/series/SH.STA.STNT.ZS)\]
```
Comment (lucasrodes) [inline etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml:61]: What's the purpose of the `\[`? Is it supposed to render differently on the data page?


image

File: etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        presentation:
+          title_public: Share of children who are stunted
+          # title_variant: UNICEF/WHO
+          attribution: UNICEF, World Health Organization and World Bank
```
Comment (lucasrodes) [inline etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml:65]: Separate different sources using `;`. See [docs](https://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/#variablepresentationattribution).

**Review by lucasrodes — CHANGES_REQUESTED** [review]
minor things

**Comment by Marigold** [comment]
Thanks Lucas, fixed all issues and merging...

**Pull request merged**